### PR TITLE
Fix ApproxFunBase upper bounds

### PR DESCRIPTION
**Pull request by dlfivefifty**

### CODE REVIEW

File: A/ApproxFunBase/Compat.toml [hunk 1]
```diff
@@ -1,54 +1,51 @@
 ["0-0.0.1"]
-RecipesBase = "0.5-0"
```
Comment (fredrikekre) [inline A/ApproxFunBase/Compat.toml]: Why do you need to change e.g. this? There is not RecipesBase 1? Similar on the other changes

Comment (dlfivefifty) [reply A/ApproxFunBase/Compat.toml]: Ah good point. To be honest I thought `-*` and `-0` meant the same thing...

Comment (fredrikekre) [reply A/ApproxFunBase/Compat.toml]: `-0` means "up to 1"

Comment (dlfivefifty) [reply A/ApproxFunBase/Compat.toml]: Ok I’ll fix it

**Pull request merged**